submissions/benjaminKamena/JavaScript/index.js

```diff
@@ -28,3 +28,41 @@ for(let i = 0; i< productList.length; i++){
 }
 
 //TASK 2
+
+// Writing the numbers from 1 and the positive difference between 7 and 123 
+var output = Math.abs(7-123);
+function diffNum(){
+    for(var i = 1; i <= output; i++){
+        if(i % 3 === 0 && i % 5 === 0){
+            return "Software Developer";
+        }else if(i % 3 === 0){
+            return "Software";
+        }else if(i % 5 === 0){
+            return "Developer";
+        } else{}
```

Comment:
log out `i` in your `else` block. Also, don't use `return`. It will stop other code execution ones it reaches a `return` statement